TIGLViewer/src/TIGLViewerSettingsDialog.cpp

```diff
@@ -25,13 +25,17 @@
 #include <cmath>
 #include <iostream>
 
+#include "TIGLViewerMaterials.h"
+
 #define WORST_TESSELATION 0.01
 #define BEST_TESSELATION 0.00001
 
 #define WORST_TRIANGULATION 0.01
 #define BEST_TRIANGULATION 0.00005
 
 #define BTN_STYLE "#buttonColorChoser {background-color: %1; color: black; border: 1px solid black; border-radius: 5px;} #buttonColorChoser:hover {border: 1px solid white;}"
+#define BTN_SHAPE_STYLE "#buttonShapeColorChoser {background-color: %1; color: black; border: 1px solid black; border-radius: 5px;} #buttonShapeColorChoser:hover {border: 1px solid white;}"
```

Comment:
The style seems to be the same for all buttons, we do you need to define multiple styles here?

Reply:
Yes probably we could use one style with other arguments? I am not so familiar with QT. The named object in the style is different. So I thought every objects needs its own style? since bg-color attribute is changed individually for each of them. I will try to figure out a way.

Reply:
Do you have a nice idea how we could solve this issue?

Reply:
Just replace the 3 lines using:

```cpp
#define BTN_STYLE "#%2 {background-color: %1; color: black; border: 1px solid black; border-radius: 5px;} #%2:hover {border: 1px solid white;}"
```

And the later in the updateBGColorButton use it like

```cpp
QString qss = QString(BTN_STYLE).arg(_bgcolor.name(), "buttonColorChoser");
```

or 

```cpp
QString qss = QString(BTN_STYLE).arg(_bgcolor.name(), "buttonShapeColorChoser");
```

and so on...

Reply:
Some background infos: https://stackoverflow.com/a/17429350

Reply:
Thanks! I added it in my last commit. It works like a charm and looks clean!